blog/report.md

```diff
@@ -0,0 +1,71 @@
+# Benchmark Report
+
+## Goal
+We want to give the user the opportunity to know the minimum size of OCP cluster that can stand for his daily logs written and read from Loki. Our goal is to create a table that specifies for each "T-shirt" size the average and max daily read and write capabilities of Loki.
```

Comment:
s/minimum size of OCP cluster/recommended LokiStack configuration

explain --> S/XXX/YYY stands for replacing XXX with YYY

Reply:
s/OCP/OpenShift